docs/configure-archive-logs.md

```diff
@@ -59,3 +61,25 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Suggested alternatives
+
+Argo Workflows log storage facilities are quite basic.
+
+We recommend using a fully featured Kubernetes aware logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+
+We also recommend adding [links](links.md) to connect from the Argo UI to your logging facility's UI. Examples are given in the [workflow-controller-configmap.yaml](workflow-controller-configmap.yaml).
```

Comment:
```suggestion
You can add [links](links.md) to connect from the Argo UI to your logging facility's UI. See examples in the [`workflow-controller-configmap.yaml`](workflow-controller-configmap.yaml).
```

- shorter / more direct and improved tense, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices)
- backticks

Reply:
Done